docs/howto/tracing.md

```diff
@@ -1,16 +1,57 @@
-(trace-the-charm-code)=
-# Trace the charm code
+(how-to-tracing)=
+# How to trace charm code
+## Introduction
 
-`ops[tracing]` provides the first party charm tracing library,
-`ops.tracing.Tracing`, allowing you to observe and instrument your charm's
-execution using OpenTelemetry.
+This document describes how to trace your charm code and send the trace data to the [Canonical Observability Stack](https://documentation.ubuntu.com/observability/).
 
-Refer to the {ref}`ops_tracing` reference for the canonical usage example, configuration
-options, and API details.
+### Why trace your charms
+
+Observability transforms a Juju deployment from a black box to a real-time system.
+
+Trace data is structured and contextual; {ref}`this helps users <tracing-users>` understand the application's behaviour at different points in your charm's lifecycle.
+
+### What is traced
+
+The responsibility to instrument the Python code is divided between Ops, charm libraries and charms.
+
+#### What Ops and charm libraries provide
+
+Ops contains its own instrumentation, as well as unit and integration tests for it.
 
-## Getting started
+- Ops creates the root span and a separate span when calling each observer, and these can be relied on in your tests.
```

Comment:
I'm not sure what "relied on" means here. Do you mean that the spans are available for inspection/assertion in the tests? If so, I think a different phrase would be clearer. ", and these are collected during tests" or something like that?

Reply:
rewritten.